qemu/tests/qemu_guest_agent.py

```diff
@@ -1358,7 +1358,12 @@ def get_new_disk(disks_before_plug, disks_after_plug):
             self.vm.devices.simple_unplug(devs[0], self.vm.monitor)
         finally:
             if self.gagent.get_fsfreeze_status() == self.gagent.FSFREEZE_STATUS_FROZEN:
-                self.gagent.fsthaw(check_status=False)
+                try:
+                    self.gagent.fsthaw(check_status=False)
+                except guest_agent.VAgentCmdError as detail:
+                    if not re.search("fsfreeze is limited up to 10 seconds", str(detail)):
```

Comment:
That means it is acceptable when fsthaw failed with message `fsfreeze is limited up to 10 seconds`, right?

Reply:
yes.